src/pages/registration.tsx

```diff
@@ -43,35 +50,105 @@ const InteractionsContainer = styled.div`
   width: 100%;
 `;
 
+const DropIcon = styled.img`
+  width: 4rem;
+`;
+
+interface UserForm {
+  firstName: string;
+  lastName: string;
+  gender: string;
+}
+
+interface FormEntry {
+  field: string;
+  value: string;
+}
+
+const emptyForm: UserForm = {
+  firstName: '',
+  lastName: '',
+  gender: ''
+};
+
+const formReducer = (state: UserForm, { field, value }: FormEntry) => ({
+  ...state,
+  [field]: value,
+});
+
 function Registration() {
   const [step, setStep] = useState(0);
+  const [userForm, setInfo] = useReducer(formReducer, emptyForm);
+
+  const handleFormChange = (e: any) => {
+    const name = e.target.name;
+    const value = e.target.value;
+    setInfo({ field: name, value: value });
+  };
+  
+  const handleDropdown = (name: string, value: string) => {
+    setInfo({ field: name, value: value });
+  };
 
   const personalInformation = (
     <FormContainer>
       <h3>Personal information</h3>
+      <form style={{ width: '100%' }}>
+        <Input
+          onChange={handleFormChange}
+          type='text'
+          name='firstName'
+          value={userForm.firstName}
+          placeHolder='first name'
+        ></Input>
+        <Input
+          onChange={handleFormChange}
+          type='text'
+          name='lastName'
+          value={userForm.lastName}
+          placeHolder='last name'
+        ></Input>
+        <Dropdown 
+          name='gender' 
+          items={[
+            'one',
+            'two',
+            'three',
+            'four'
+          ]}
```

Comment:
These should be substituted for actual values.

Reply:
The four options should be:
* they/them
* he/him
* she/her
* other (text input)

Reply:
Replaced with education for now